### PR TITLE
[BG] optimisation to code

### PR DESCRIPTION
**Pull request by BossNeo**
1. Merge C and F in short format
2. Took out < lampi_brightness > words forbrightness for more clean view of code
3. Add more native words to set up lights
4. Adding sentece to test the new words

### CODE REVIEW

File: sentences/bg/_common.yaml [hunk 1]
```diff
@@ -51,11 +51,13 @@ expansion_rules:
...
   all: "(всички[те] | всяк(о|а) | всеки)"
   lampi: "(ламп(а|ата|и|ите) | осветление[то] | крушк(а|ата|и|ите))"
+  lampi_brightness: "( [на] [яркост[та]] [на] [светене[то]] [на] [светлина[та]])"
   temp: "(температур(а|ата|ите) | градус[а|и|те] | термостат[а|ът|ите])"
+  cover: "(щор(а|и|ата|ите) |завес(a|и|ата|ите) | перде[то|та|тата] | кепенци[те] | капа(к|ка|кът|ци|ците))"
   temperature: "{temperature} [градуса] [{temperature_unit}]"
   turn_on: (пусни | включи | светни | запали)
   turn_off: (спри | изключи | изгаси | угаси)
-  set: (направи | сложи | постави | смени | промени | настрой | задай | нагласи | тури)
+  set: (направи | сложи | постави | смени | промени | настрой | задай | нагласи | тури | увеличи | намали )
```
Comment (hristo-atanasov) [inline sentences/bg/_common.yaml:60]: `увеличи` и `намали` не са подходящ превод за `set`

**Review by hristo-atanasov — CHANGES_REQUESTED** [review]
Освен другите коментари, които съм ти написал, можеш да смениш  на всички места `lampi` с `lights` ..

File: sentences/bg/_common.yaml (hunk 1 above)
Comment (BossNeo) [reply sentences/bg/_common.yaml:60]: Ok agree Tommorow will fax all that and will add set max functionality. Sorry for the delay

**Comment by hristo-atanasov** [comment]
You should fix the conflicts.

**Comment by BossNeo** [comment]
> Only those with [write access](https://docs.github.com/articles/what-are-the-different-access-permissions) to this repository can merge pull requests. 
The button is not active and the conflict acure from MAIN file . I dont know who touch it and left it with error. For that my branch now is not working. There is no problem with this branch at all . 
sentences/bg/cover_HassCloseCover.yaml
sentences/bg/cover_HassOpenCover.yaml
must to be deleted, but that dont solve the problem from main. 
Fix main and delete this request i will make new if need.

**Comment by BossNeo** [comment]
i create new PR with all fix

**Pull request closed without merging**